book/src/configuration.md

```diff
@@ -150,6 +150,8 @@ All git related options are only enabled in a git repository.
 | Key | Description | Default |
 |--|--|---------|
 |`hidden` | Enables ignoring hidden files. | true
+|`follow-links` | Follow symlinks instead of ignoring them | true
+|`deadup-links` | Ignore symlinks that point at files already shown in the picker | true
```

Comment:
I think this should be "dedup", no? I suppose it could be "deduplicate-links" to be verbose

Reply:
Sorry my spelling isn't always the best :/ I like the sound of `deduplicate-links` :+1:  although I kept `dedup` in the code. That abbreviation is common enough in code I think (rust stdlib uses it)